### ISSUE TITLE
Premium Content: Paid flag in the block inserter

### ISSUE THREAD

**Issue by artpi** [Automattic/wp-calypso#41584]
All the blocks that require upgrading have a paid flag in the block inserter. We need to add one to the Premium content block.

**Comment by artpi** [comment]
![obraz](https://user-images.githubusercontent.com/3775068/80577896-f18a5380-8a07-11ea-84bf-0ac7e2c3ea73.png)

**Comment by artpi** [comment]
(@fditrapani ) Just wanted to note that the flag does not show for the other blocks if you're on a paid plan that supports the button.

**Comment by fditrapani** [comment]
Sounds good. Thanks @artpi. There's work going on to update the design for this. We should do it in such a way that we can inherit what ever they do.

**Comment by n3f** [comment]
@fditrapani, should we wait on this?  Is it better in the mean time to just add the `(paid)` label like the existing blocks?

**Comment by fditrapani** [comment]
It could take a while for the other work to get implemented so we should move ahead on this if we want it in before the public announcement.

**Comment by n3f** [comment]
Looking this morning, I don't see any paid flags:

![image](https://user-images.githubusercontent.com/6354169/81597015-dc071900-9379-11ea-9d2e-62969c35b29d.png)

@artpi, do you know anything about this?  Is the paid flag no longer needed?

**Comment by fditrapani** [comment]
@n3f do you know if you're on a paid plan for that site? Artur [pointed out](https://github.com/Automattic/wp-calypso/issues/41584#issuecomment-6210753040) the flags don't show if you're on a paid plan that supports the feature.

**Comment by n3f** [comment]
Ahh, that's it.
